beacon-chain/p2p/handshake.go

```diff
@@ -155,7 +154,8 @@ func (s *Service) AddConnectionHandler(reqFunc func(ctx context.Context, id peer
 func (s *Service) AddDisconnectionHandler(handler func(ctx context.Context, id peer.ID) error) {
 	s.host.Network().Notify(&network.NotifyBundle{
 		DisconnectedF: func(net network.Network, conn network.Conn) {
-			log := log.WithField("peer", conn.RemotePeer().Pretty())
+			multiAddr := fmt.Sprintf("%s/p2p/%s", conn.RemoteMultiaddr().String(), conn.RemotePeer().String())
```

Comment:
This is duplicated logic here and in the connection handler.

```go
multiAddr := fmt.Sprintf("%s/p2p/%s", conn.RemoteMultiaddr().String(), conn.RemotePeer().String())
```

It would be ideal to abstract this into a common function that can be used between these two handlers, something like
```go
func peerMultiaddrString(conn network.Conn) string {
  return fmt.Sprintf("%s/p2p/%s", conn.RemoteMultiaddr().String(), conn.RemotePeer().String())
}
```

Reply:
I will work on this.  Thanks

Reply:
Just pushed a commit to address this.